domain/secret/state/state.go

```diff
@@ -314,15 +314,14 @@ func (st State) createSecret(
 		}
 	}
 
-	now := time.Now()
+	now := time.Now().UTC()
 	dbSecret := secretMetadata{
-		ID:          uri.ID,
-		Version:     version,
-		Description: secret.Description,
-		AutoPrune:   secret.AutoPrune,
-		CreateTime:  now,
-		UpdateTime:  now,
+		ID:         uri.ID,
+		Version:    version,
+		CreateTime: now,
+		UpdateTime: now,
```

Comment:
Why do we remove the Description and AutoPrune?

Reply:
The non core attributes are set in `updateSecretMetadataFromParams`.
This will also be used when Update is implemented.